src/components/Tool/FixPhoneNumbers/Contact.tsx

```diff
@@ -295,7 +311,7 @@ const Contact: React.FC<Props> = ({
                     action={
                       <Button
                         data-testid={`confirmButton-${person.id}`}
-                        onClick={() => handleSingleConfirm(person, numbers)}
+                        onClick={() => handleSubmit()}
```

Comment:
If I'm reading this correctly, this oes the same thing as `handleSingleConfirm(person, numbers)`

Reply:
Sorry, I don't think I was clear. I wasn't requesting for it to change, just confirming that it mostly does the same thing.